src/test/common/page/guidance.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const Page = require('./page')
+
+const { PAGES, getPageMetadata } = require('../../../web/routes/guidance')
+
+/**
+ * Page object for all the guidance pages. This may extend Page, but as it represent multiple static page, it must
+ * implement its own methods for navigation, but can use all the find methods from Page when required.
+ */
+class Guidance extends Page {
+  async openGuidancePage (baseURL, pageName, lang = 'en') {
+    try {
+      await this.openPage(`${baseURL}${this.getPageWithTitle(PAGES, pageName).path}`, lang)
+      return super.waitForPageWithTitle(this.getPageTitle(pageName))
+    } catch (error) {
+      console.error(`Error caught trying to open guidance page at: ${baseURL}`, error)
+      throw new Error(error)
+    }
+  }
+
+  getPageTitle (pageName) {
+    return 'GOV.UK - ' + pageName
```

Comment:
Could be:
```
return `GOV.UK - ${pageName}`
```

as you've used this format on above lines

Reply:
Done.